Modules/Filtering/ImageIntensity/test/itkTernaryMagnitudeImageFilterTest.cxx

```diff
@@ -133,14 +133,14 @@ itkTernaryMagnitudeImageFilterTest(int argc, char * argv[])
   // Create the filter
   TernaryMagnitudeImageFilterType::Pointer filter = TernaryMagnitudeImageFilterType::New();
 
-  ITK_EXERCISE_BASIC_OBJECT_METHODS(filter, TernaryMagnitudeImageFilter, TernaryFunctorImageFilter);
+  ITK_EXERCISE_BASIC_OBJECT_METHODS(filter, TernaryMagnitudeImageFilter, TernaryGeneratorImageFilter);
 
   // Set the input images
   filter->SetInput1(inputImageA);
   filter->SetInput2(inputImageB);
   filter->SetInput3(inputImageC);
 
-  filter->SetFunctor(filter->GetFunctor());
+  // filter->SetFunctor(filter->GetFunctor());
```

Comment:
Why is this commented out? Maybe remove?

Reply:
The "GetFunctor" method is not available in the Generator style classes. This method was also removed from the classes derived from the unary and binary generator base classes.